docs/data/joy/components/input/input.md

```diff
@@ -166,6 +166,24 @@ You can use these to customize the component with both the `sx` prop and the the
 
 {{"demo": "InputVariables.js", "hideToolbar": true}}
 
+## Third-party library integration
```

Comment:
I think this header is more clear—"library integration" feels vague, when this is really about integrating specifically with formatting libs. 

I would also suggest making this an h3 (and headers below it h4s), and moving it up into the ## Customization section (maybe at the bottom of it).

```suggestion
### Third-party formatting
```